storage_tiering.cpp

```diff
@@ -797,12 +849,13 @@ namespace irods {
                                         config_.group_attribute,
                                         _object_path,
                                         source_replica_number);
-            const auto low_tier_resource_name = get_restage_tier_resource_name(
-                                                    comm_,
-                                                    group_name);
-            // do not queue movement if data is on minimum tier
+            const auto low_tier_resource_name = get_restage_tier_resource_name(comm_, group_name);
+            const auto source_resource_tier = get_group_tier_for_resource(comm_, _source_resource, group_name);
+            const auto low_tier_resource_tier = get_group_tier_for_resource(comm_, low_tier_resource_name, group_name);
+
+            // do not queue movement if data is on minimum tier or lower
             // TODO:: query for already queued movement?
-            if(low_tier_resource_name == _source_resource) {
+            if (source_resource_tier <= low_tier_resource_tier) {
                 return;
```

Comment:
Perhaps we should add a trace/debug message for the admin here?

Reply:
Done. Used old logger for consistency. There's already an issue for migrating to the new logger: #244